scribe/src/main/scala/io/chrisdavenport/log4cats/scribe/ScribeLogger.scala

```diff
@@ -0,0 +1,44 @@
+package io.chrisdavenport.log4cats.scribe
+
+import io.chrisdavenport.log4cats.Logger
+import scribe.{Logger => Base}
+import cats.effect.Sync
+
+object ScribeLogger {
+
+  def fromLogger[F[_]: Sync](logger: Base): Logger[F] = new Logger[F]{
+    // Law Abiding But Not What We Want
+    def isTraceEnabled: F[Boolean] = 
+      Sync[F].delay(???)
```

Comment:
@DavidGregory084 Is there any ability for this functionality in scribe somewhere I'm missing. We may need to build a multi-level algebra for universality is scribe can't support these operations.